tests/erlang_tests/test_refc_binaries.erl

```diff
@@ -106,7 +106,7 @@ test_send() ->
     Bin = create_binary(1024),
     ?VERIFY(MemoryBinarySize + 1024 =< erlang:memory(binary)),
 
-    Pid = erlang:spawn(fun() -> loop(#state{}) end),
```

Comment:
So I guess I like the fact that this makes the undecorated spawn calls consistent in all the tests, but I do also wonder if it is a good pattern to prefix erlang functions with the `erlang:` module prefix.  The nit picker in me wants this, but I don’t know what others think.  This is more of a side comment, and not suggesting we change the world in this PR.

Reply:
These functions are auto-imported.
erlang:func syntax should be limited to non-auto imported functions (which are documented with erlang: prefix in erlang man page).

See https://github.com/erlang/otp/blob/7388d5ad2903a4722ff76d6bb10abf1fda7f3cfb/erts/doc/src/erlang.xml#L35